packages/paste-core/components/progress-steps/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@twilio-paste/progress-steps",
+  "version": "0.0.0",
+  "category": "data display",
+  "status": "production",
+  "description": "Progress steps show a user a clear path to complete a complex multi step task.",
```

Comment:
what about: Progress Steps can be a presentational or interactive component and shows users an outline of a complex multi-step task.

take it or leave it